database/gov.go

```diff
@@ -220,27 +230,28 @@ func (db *Db) SaveDeposits(deposits []types.Deposit) error {
 	var accounts []types.Account
 	for i, deposit := range deposits {
 		vi := i * 5
+
+		accounts = append(accounts, types.NewAccount(deposit.Depositor))
+
 		query += fmt.Sprintf("($%d,$%d,$%d,$%d,$%d),", vi+1, vi+2, vi+3, vi+4, vi+5)
 		param = append(param, deposit.ProposalID,
 			deposit.Depositor,
 			pq.Array(dbtypes.NewDbCoins(deposit.Amount)),
 			deposit.Timestamp,
 			deposit.Height,
 		)
-		accounts = append(accounts, types.NewAccount(deposit.Depositor))
 	}
 
-	// Store the depositor account
+	// Store depositors accounts
 	err := db.SaveAccounts(accounts)
 	if err != nil {
-		return fmt.Errorf("error while storing depositor account: %s", err)
+		return fmt.Errorf("error while storing depositors accounts: %s", err)
 	}
 
 	query = query[:len(query)-1] // Remove trailing ","
 	query += `
 ON CONFLICT ON CONSTRAINT unique_deposit DO UPDATE
-	SET amount = excluded.amount,
```

Comment:
Why do we ignore amount update? :o

Reply:
The amount here is ignored because with the current flow we should store all proposals deposits, instead of storing just the latest one, hence we shouldn't update the amount value here